### ISSUE TITLE
Add Pine A64 board support

### ISSUE THREAD

**Issue by dragetd** [ubitux/boards-gcase#1]
I am working on adding the specs for the pine a64 board.

**Comment by ubitux** [comment]
Cool, I don't have that one :)

I've let this project on hold due to other projects taking over, and also because I'm waiting for kernel 4.15 before I can migrate my current odroid-c2 to the nanopi neo 2 (they need 4.15 for the ethernet support) and then design its board.

**Comment by dragetd** [comment]
I have an odroid Zero, RPi 1, RPi Zero and eventually a few others to add. :)

Tho kind of busy with things, might take a little before I get to all of these.

**Comment by ubitux** [comment]
I have the RPi 1 on my TODO list, not sure how long it will take, please leave it to me ;)

**Comment by dragetd** [comment]
Sure, I'll leave the RPi1 to you. :) But I might give pi zero a shot, ok? :)

Concerning the Pine64, this is how far I am right now: https://github.com/dragetd/shimonbox/tree/patch_pine64_b

It still lacks several things:

- [ ] Board color
- [ ] GPIO Header x2
- [ ] Expansion Header
- [ ] LCD/Camera bus
- [ ] (Optional) Reset-Button
- [ ] (Optional) LED holes
- [ ] Picture of printed case

I did print the current version and it worked pretty well. It had very minor asymmetries and the 'worst' thing were the mounting holes slightly off (required some force to insert there). But I corrected these things and it should print just fine. Tho before printing another one, I would like to tackle some of the TODOs.

**Comment by ubitux** [comment]
> But I might give pi zero a shot, ok? :)

Sure, I don't have that one

> Board color

see `clr` parameter in `extrude_plate()`

> GPIO Header x2

Female and pin headers in `electronics/header.scad` don't do the trick?

**Comment by dragetd** [comment]
Yes it does. I was just too lazy, since I do not need those headers for the case I just printed. :P I am busy with other things… once I got my other projcets sorted, I will continue here. Just thought I could already share the progress. :)

**Comment by dragetd** [comment]
Please excuse my radio silence. #life happened. :( I hope I will eventually come back to progressing on this - but if anyone else wants to continue the work, please feel free!
